src/nupic/algorithms/Cell.hpp

```diff
@@ -166,6 +166,7 @@ namespace nupic {
           }
 
           // Sort segments according to activation frequency
+          // ZZZ Where is the sort?
```

Comment:
Don't know and can't find the history for the file in the archived repos. I'm not sure it is worth looking into changes but you could leave a TODO comment to investigate.